transfer/hashgrid.hpp

```diff
@@ -0,0 +1,39 @@
+#ifndef MFEM_L2P_HASH_GRID_HPP
+#define MFEM_L2P_HASH_GRID_HPP
+
+#include <vector>
+#include "../fem/fem.hpp"
+#include "box.hpp"
+
+
+
+namespace mfem {
+
```

Comment:
It would be good to add a short explanation what these classes (all new classes) are used for. Also it would be good to explain why they must be in MFEM.

Reply:
ping @zulianp

Reply:
I am currently away for the Swiss army COVID response. I will probably be back in January and I will have time then to look at the comments. Cheers!

Reply:
take care Patrick!

Reply:
It did not make sense to be in MFEM. The class has been removed.